src/@orb.yml

```diff
@@ -1,4 +1,4 @@
-version: 2.1
+version: 2.2
```

Comment:
This is not the version of our Orb but the version of the Orb Platform we are using in our Pipeline that tests and releases our Orb. I think it is the latest version they have so switch this back to `2.1`.

Ready more about it: https://circleci.com/docs/2.0/orb-intro/

The version bump of our Orb is automated, you just need to set the git subject to include `[semver:patch|minor|release|skip]` to indicate the type of release. This is documented at
the https://github.com/lacework/circleci-orb-lacework#how-to-contribute